### ISSUE TITLE
Run bin/estimate_box_runtime.jl on every quarter square degree of the sky

### ISSUE THREAD

**Issue by jeff-regier** [jeff-regier/Celeste.jl#425]
Might as well do this upfront using taskfarmer. The current version of `estimate_box_runtime.jl` fits a psf and interpolates the sky background noise for each image. Both are unnecessary for estimating runtime. Some restructuring of the image loading routine is needed could make `estimate_box_runtime.jl` faster.

**Comment by jeff-regier** [comment]
Done! To reduce runtime and memory consumption, I estimated runtime with a metric different than number of active pixels. Instead, I computed a "hardness" score by multiplying the number of sources to optimize by the number of RCFs in each box. Since there's an upper bound on the of active pixels per source-rcf pair this is not a bad approximation, and it's much faster. Not every source appears in every RCF of course---multiplying by the number of RCF just scales the hardness to account for the average density of images.

The total hardness units for all of SDSS is 9,621,754,068. The most difficult quarter degree is in stripe 82. It has 2,773,047 hardness units --- about 0.2%, or 1/3500, of the overall work. If we run more than 3500 processes, that task would be a bottleneck. (We can drop down to .25x.25 degree boxes if necessary.) If we exclude stripe 82, the most difficult task has 1,237,428 hardness.

I posted the all the task difficulties [here](https://www.dropbox.com/s/h1nscqr54649mif/quarter_degrees_sorted_by_difficulty.txt?dl=0).
